requirements.txt

```diff
@@ -1,27 +1,57 @@
-six
```

Comment:
Avoid versions for now, this is boilerplate application will be updated with the latest versions.